doc/developers_guide.md

```diff
@@ -5,3 +5,5 @@ This guide contains information for developers of this adapter.
 ## Release
 
 Release this adapter according to this [tutorial](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-template-publishing-applications.html).
+
+Publishing of this repository will fail, due to a [bug in SAM](https://github.com/aws/serverless-application-model/issues/1931). As a workaround, remove the `RetryPolicy` from `sam/template.yaml`, package and publish, then add it again and package and publish again.
```

Comment:
```suggestion
Publishing of this repository will fail, due to a [bug in SAM](https://github.com/aws/serverless-application-model/issues/1931). As a workaround, remove the `RetryPolicy` from `sam/template.yaml`, package and publish, then deploy it again.
```

Reply:
Is this what you meant? The last "package and publish again" confusing.

Reply:
I tried to clarify